packages/config/config/devices/0x001a/rf9501.json

```diff
@@ -103,6 +103,30 @@
 					"value": 2
 				}
 			]
+		},
+		{
+			"#": "10",
+			"label": "Notify Accessory",
+			"description": "Ensures that changes to the master node automatically notify accessory switches",
```

Comment:
We tend to rename config params to make them easier to understand. How about this?
```suggestion
			"label": "Forward Z-Wave Commands",
			"description": "Keep associated accessory switches in sync",
```